src/constants/DELIVERY_INFOS.ts

```diff
@@ -0,0 +1,7 @@
+import type { DeliveryInfo } from '../types/DeliveryInfo.type';
+
+export const DELIVERY_FEE_DISCOUNT_THRESHOLD = 100000;
+
+export const DELIVERY_INFOS = {
+  ISLANDS_AND_MOUNTAINS: { title: '제주도 및 도서 산간 지역', surcharge: 3000 } as DeliveryInfo,
+};
```

Comment:
데이터를 선언할 때는 타입 단언보다는 명시적 선언을 권장드립니다. 또한 이 데이터가 추가 배송료가 붙는 지역에 대한 정보를 가진 데이터임을 알 수 있도록 이름의 구체화가 좀 더 필요하지 않을까 싶습니다. 

```suggestion
export const DELIVERY_INFOS: Record<string, DeliveryInfo> = {
  ISLANDS_AND_MOUNTAINS: { title: '제주도 및 도서 산간 지역', surcharge: 3000 } ,
};
```

Reply:
타입 선언보다는 단언이 타입 추적과 확인에 훨씬 도움이 되겠네요 😄
타입을 단언해 주었습니다!

상수 이름의 경우에는, 추가 배송비가 붙는 지역 외에도 퀵 배송, (대형 가전 혹은 가구의 경우에는) 방문 설치 등 조금 더 다양한 조건이 붙을 수 있다고 생각해 DELIVERY_INFOS라는 다소 포괄적인 이름을 붙였어요!

이를 고려해 미션 요구사항에서는 제주도 및 도서산간의 조건 하나밖에 없었지만 일부러 객체 모음으로 만들어 관리할 수 있게 구현하기도 했습니다.

Reply:
> 타입 선언보다는 단언이 타입 추적과 확인에 훨씬 도움이 되겠네요 😄 타입을 단언해 주었습니다!

제가 말씀 드린 의도는 반대입니다 ㅎㅎ 사실 이 맥락에서는 단언, 선언 둘 다 상관 없지만 잘못 추론되는 타입을 강제 변환하는 목적이 아닌 구현체에 타입을 부여하는 목적에서는 타입 선언이 좀 더 적절할 듯 하여 드린 말씀입니다.